err.go

```diff
@@ -184,6 +185,53 @@ func With(err error, kvs ...any) *Err {
 	return e.With(kvs...)
 }
 
+// WithTrace sets the error trace to a certain depth.
+// Error traces are already generated for the location
+// where clues.Wrap or clues.Stack was called.  This
+// call is for cases where Wrap or Stack calls are handled
+// in a helper func and are not reporting the actual
+// error origin.
+func (err *Err) WithTrace(depth int) *Err {
```

Comment:
this and the other `WithTrace` function needs hints about what to set depth to if there's no default easily available. I'm not sure what to pass and determining that would require some poking around/experimentation

Reply:
Ah, good call, will add.